static/js/pages/instances/actions/OpenVgaBtn.tsx

```diff
@@ -25,11 +25,11 @@ const OpenVgaBtn: FC<Props> = ({
     if (instance.status !== "Running") {
       return "Instance must be running";
     }
-    return undefined;
+    return "VGA Console";
   };
 
   return (
-    <Tooltip message={getTooltipMessage()} position="btm-center">
+    <Tooltip message={getTooltipMessage()} position="top-right">
```

Comment:
This should be the `OpenConsoleBtn` now, that links to either the text console (for containers) or the graphical console (for virtual machines).

Reply:
There is no `OpenConsoleBtn` now and it would not be the right timing to create it since the VD of the instance detail page are still pending validation. Once this get clarified, it can be updated here as well, but out of the scope of this PR I'd say since it depends on a design that is still pending approval. Same applies to the instance actions probably, if we want to switch to the "icons only" view, I'd keep them as they are (namely, only the start/stop button) until the VD gets updated, and take care of the changes in a later PR.